src/charm.py

```diff
@@ -226,6 +228,25 @@ def main(self, event):
 
         self.model.unit.status = ActiveStatus()
 
+    def log_owasp_events(self, event):
+        owasp_logger = OWASPLogger(appid="dex.owasp-logger")
```

Comment:
suggestion (non-blocking): should not we create the logger in init as we do for normal logger ?

Reply:
I think Michal has a point, even when we re-write to chisme components we'd define the logger (and `set_default`) in the init of the component, while the logging would be handled in a separate function. At least this is the pattern we generally implement.